Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPage.mm

```diff
@@ -913,4 +913,44 @@ static void swizzledPerformTextSearchWithQueryString(id, SEL, NSString *, UIText
     EXPECT_TRUE(findSessionSupportsReplacement());
 }
 
+static bool hasPerformedTextSearchWithQueryString = false;
+
+static void swizzledPerformTextSearchInPDFWithQueryString2(id, SEL, NSString *, UITextSearchOptions *, id<UITextSearchAggregator> aggregator)
+{
+    [aggregator finishedSearching];
+    hasPerformedTextSearchWithQueryString = true;
+}
+
+TEST(WebKit, WKPDFViewFindInteractionAfterPageRefresh)
```

Comment:
`FindInPDFAfterReload`? This would match then naming scheme of the other find-in-PDF test.